src/Main.java

```diff
@@ -1,5 +1,62 @@
 public class Main {
     public static void main(String[] args) {
+        System.out.println("Задача 1");
+        for (int i = 1; i <= 10; i++) {
+            System.out.print(i + "");
+        }
+        System.out.println();
+        System.out.println("Задача 2");
+        for (int i = 10; i >= 1; i--) {
+            System.out.print(i + "");
 
+        }
+        System.out.println();
+        System.out.println("Задача 3");
+        for (int i = 0; i <= 17; i++) {
+            if (i % 2 == 0)
+                System.out.print(i + "");
+        }
+        System.out.println();
+        System.out.println("Задача 4");
+        for (int i = 10; i >= -10; i--) {
+            System.out.println(i + "");
+        }
+        System.out.println();
+        System.out.println("Задача 5");
+        for (int year = 1904; year <= 2096; year += 4) {
+            System.out.println(year + "Високосный год");
+        }
+        System.out.println();
+        System.out.println("Задача 6");
+        for (int i = 7; i <= 98; i += 7) {
+            System.out.println(i + "");
+        }
+        System.out.println();
+        System.out.println("Задача 7");
+        for (int i = 2; i <= 512; i *= 2) {
+            System.out.println(i + "");
+        }
+        System.out.println();
+        System.out.println("Задача 8");
+        int amount = 29_000;
+        int sum = 0;
+        for (int month = 1; month <= 12; month++) {
+            sum += amount;
+            System.out.println("Месяц" + month + "сумма накоплений равна" + sum + "рублей");
+        }
+        System.out.println();
+        System.out.println("Задача 9");
+        sum = 0;
+        double percent = 1 / 100;
```

Comment:
деление целочисленное, у тебя в percent будет 0, а не 0.01. Чтобы было вещественное деление, надо привести хотя бы один из операндов деления к вещественному числу
```suggestion
        double percent = 1D / 100;
```